test/tesla/adapter/finch_test.exs

```diff
@@ -0,0 +1,16 @@
+defmodule Tesla.Adapter.FinchTest do
+  use ExUnit.Case
+
+  @finch_name MyFinch
+
+  use Tesla.AdapterCase, adapter: Tesla.Adapter.Finch, adapter_opts: [name: @finch_name]
```

Comment:
We could use the same syntax as in config:

```
use Tesla.AdapterCase, adapter: {Tesla.Adapter.Finch, [name: @finch_name]}
```

Reply:
Done!